addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1780,6 +1780,22 @@ options.registry.ScrollButton = options.Class.extend({
     },
 });
 
+options.registry.ImageCaption = options.Class.extend({
+    defaultCaption: _t('Write a caption under your image'),
```

Comment:
You should use `_lt` when you want to translate at the first level of a JS module. Or move that string in the constructor of the options.

Reply:
thanks for the heads up, I'll know for next time (as noted above, the feature has been dropped as discussed with FP)